code/src/pages/MovieBio.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState, useEffect } from "react";
+import { useParams, Link, useHistory } from "react-router-dom";
+
+import { BackButton } from "components/js/BackButton";
```

Comment:
Nice to have the back button as a separate component, if I ever implement it I will checkout your approach for this!